code/controllers/subsystems/traumas.dm

```diff
@@ -53,7 +53,7 @@ var/datum/controller/subsystem/traumas/SStraumas
 
 					   "security"  = typecacheof(list(/obj/item/clothing/under/rank/security, /obj/item/clothing/under/rank/warden, /obj/item/clothing/head/beret/sec,
 											 	 /obj/item/clothing/under/rank/head_of_security, /obj/item/clothing/under/det, /obj/item/clothing/glasses/hud/security,
-												 /obj/item/weapon/melee/baton, /obj/item/weapon/gun/energy/taser, /obj/item/weapon/handcuffs, /obj/item/clothing/glasses/sunglasses/sechud,
+												 /obj/item/weapon/melee/baton, /obj/item/weapon/gun/energy/taser, /obj/item/weapon/handcuffs, /obj/item/clothing/glasses/sunglasses/sechud/aviator,
```

Comment:
Keep the original, but add aviators to the list

Reply:
ok fine done